shinmen-coffee/src/app/layout.tsx

```
@@ -0,0 +1,135 @@
+        <meta name="format-detection" content="telephone=no" />
+      </head>
+      <body className={`${inter.className} antialiased`}>
+        <Providers>
+          <div id="root" className="min-h-screen bg-coffee-light">
+            {children}
+          </div>
+        </Providers>
+        
+        {/* Performance Monitoring */}
+        <script
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)